core-web/libs/edit-content/src/lib/fields/dot-edit-content-category-field/components/dot-edit-content-category-field-sidebar/dot-edit-content-category-field-sidebar.component.html

```diff
@@ -0,0 +1,38 @@
+<p-sidebar
+    (onHide)="this.closedSidebar.emit()"
+    [(visible)]="visible"
+    [showCloseIcon]="false"
+    #sidebar
+    data-testId="sidebar"
+    position="right"
+    styleClass="w-9">
+    <ng-template pTemplate="header">
+        <div class="flex flex-row category-field__header">
+            <button
+                (click)="sidebar.close($event)"
+                class="p-button-rounded p-button-text"
+                data-testId="back-btn"
+                icon="pi pi-angle-left"
+                pButton></button>
+            <div class="category-field__header-title" data-testId="sidebar-title">
+                {{ 'edit.content.category-field.sidebar.header.select-categories' | dm }}
+            </div>
+        </div>
+    </ng-template>
+
+    <div class="category-field__content h-full w-full">
+        <div class="category-field__left-pane flex flex-column">
+            <div class="category-field__search">Input Search</div>
+            <div class="flex-grow-1 category-field__categories">Categories</div>
+        </div>
+
+        <div class="category-field__right-pane flex flex-column">
+            <div class="category-field__selected-categories-list flex-1">Selected Categories</div>
+            <div class="category-field__actions flex justify-content-end">
+                <button class="p-button p-button-link" data-testId="clear_all-btn" pButton>
+                    {{ 'edit.content.category-field.button.clear-all' | dm }}
+                </button>
+            </div>
+        </div>
+    </div>
+</p-sidebar>
```

Comment:
Translations?

Reply:
They are only placeholders, they will disappear with the real input and the real components of categories and selected categories.